easybuild/easyconfigs/g/gompi/gompi-2015b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2015b'
+
+homepage = '(none)'
```

Comment:
http://easybuild.readthedocs.org/en/latest/eb_list_toolchains.html ?

I'm just asking, doesn't have to be changed for me..